## 95_add_spi+cacert_ca_certs.patch by martin f. krafft <madduck@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adds the SPI Inc. and CAcert.org CA certificates

Index: nss/nss/lib/ckfw/builtins/certdata.txt
===================================================================
--- nss.orig/nss/lib/ckfw/builtins/certdata.txt
+++ nss/nss/lib/ckfw/builtins/certdata.txt
@@ -29874,3 +29874,212 @@ CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_T
 CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+ 
+#
+# Certificate "Software in the Public Interest, Inc. Root CA (2008)"
+#
+CKA_CLASS CK_OBJECT_CLASS CKO_CERTIFICATE
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "Software in the Public Interest, Inc. Root CA (2008)"
+CKA_CERTIFICATE_TYPE CK_CERTIFICATE_TYPE CKC_X_509
+CKA_SUBJECT MULTILINE_OCTAL
+\060\201\274\061\013\060\011\006\003\125\004\006\023\002\125\123
+\061\020\060\016\006\003\125\004\010\023\007\111\156\144\151\141
+\156\141\061\025\060\023\006\003\125\004\007\023\014\111\156\144
+\151\141\156\141\160\157\154\151\163\061\050\060\046\006\003\125
+\004\012\023\037\123\157\146\164\167\141\162\145\040\151\156\040
+\164\150\145\040\120\165\142\154\151\143\040\111\156\164\145\162
+\145\163\164\061\023\060\021\006\003\125\004\013\023\012\150\157
+\163\164\155\141\163\164\145\162\061\036\060\034\006\003\125\004
+\003\023\025\103\145\162\164\151\146\151\143\141\164\145\040\101
+\165\164\150\157\162\151\164\171\061\045\060\043\006\011\052\206
+\110\206\367\015\001\011\001\026\026\150\157\163\164\155\141\163
+\164\145\162\100\163\160\151\055\151\156\143\056\157\162\147
+END
+CKA_ID UTF8 "0"
+CKA_ISSUER MULTILINE_OCTAL
+\060\201\274\061\013\060\011\006\003\125\004\006\023\002\125\123
+\061\020\060\016\006\003\125\004\010\023\007\111\156\144\151\141
+\156\141\061\025\060\023\006\003\125\004\007\023\014\111\156\144
+\151\141\156\141\160\157\154\151\163\061\050\060\046\006\003\125
+\004\012\023\037\123\157\146\164\167\141\162\145\040\151\156\040
+\164\150\145\040\120\165\142\154\151\143\040\111\156\164\145\162
+\145\163\164\061\023\060\021\006\003\125\004\013\023\012\150\157
+\163\164\155\141\163\164\145\162\061\036\060\034\006\003\125\004
+\003\023\025\103\145\162\164\151\146\151\143\141\164\145\040\101
+\165\164\150\157\162\151\164\171\061\045\060\043\006\011\052\206
+\110\206\367\015\001\011\001\026\026\150\157\163\164\155\141\163
+\164\145\162\100\163\160\151\055\151\156\143\056\157\162\147
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\011\000\350\216\266\311\370\052\024\050
+END
+CKA_VALUE MULTILINE_OCTAL
+\060\202\010\016\060\202\005\366\240\003\002\001\002\002\011\000
+\350\216\266\311\370\052\024\050\060\015\006\011\052\206\110\206
+\367\015\001\001\005\005\000\060\201\274\061\013\060\011\006\003
+\125\004\006\023\002\125\123\061\020\060\016\006\003\125\004\010
+\023\007\111\156\144\151\141\156\141\061\025\060\023\006\003\125
+\004\007\023\014\111\156\144\151\141\156\141\160\157\154\151\163
+\061\050\060\046\006\003\125\004\012\023\037\123\157\146\164\167
+\141\162\145\040\151\156\040\164\150\145\040\120\165\142\154\151
+\143\040\111\156\164\145\162\145\163\164\061\023\060\021\006\003
+\125\004\013\023\012\150\157\163\164\155\141\163\164\145\162\061
+\036\060\034\006\003\125\004\003\023\025\103\145\162\164\151\146
+\151\143\141\164\145\040\101\165\164\150\157\162\151\164\171\061
+\045\060\043\006\011\052\206\110\206\367\015\001\011\001\026\026
+\150\157\163\164\155\141\163\164\145\162\100\163\160\151\055\151
+\156\143\056\157\162\147\060\036\027\015\060\070\060\065\061\063
+\060\070\060\067\065\066\132\027\015\061\070\060\065\061\061\060
+\070\060\067\065\066\132\060\201\274\061\013\060\011\006\003\125
+\004\006\023\002\125\123\061\020\060\016\006\003\125\004\010\023
+\007\111\156\144\151\141\156\141\061\025\060\023\006\003\125\004
+\007\023\014\111\156\144\151\141\156\141\160\157\154\151\163\061
+\050\060\046\006\003\125\004\012\023\037\123\157\146\164\167\141
+\162\145\040\151\156\040\164\150\145\040\120\165\142\154\151\143
+\040\111\156\164\145\162\145\163\164\061\023\060\021\006\003\125
+\004\013\023\012\150\157\163\164\155\141\163\164\145\162\061\036
+\060\034\006\003\125\004\003\023\025\103\145\162\164\151\146\151
+\143\141\164\145\040\101\165\164\150\157\162\151\164\171\061\045
+\060\043\006\011\052\206\110\206\367\015\001\011\001\026\026\150
+\157\163\164\155\141\163\164\145\162\100\163\160\151\055\151\156
+\143\056\157\162\147\060\202\002\042\060\015\006\011\052\206\110
+\206\367\015\001\001\001\005\000\003\202\002\017\000\060\202\002
+\012\002\202\002\001\000\334\066\346\107\102\302\304\121\165\051
+\207\100\303\330\216\041\006\322\030\116\353\357\040\275\220\074
+\205\020\023\214\051\133\224\143\366\364\055\361\006\102\221\271
+\031\304\102\151\010\277\213\066\105\352\050\005\063\111\110\240
+\047\103\223\065\212\101\330\170\263\360\357\263\156\055\335\321
+\313\175\352\364\165\046\323\076\220\072\356\327\347\054\004\265
+\174\341\365\174\305\116\357\167\275\134\242\223\063\222\316\175
+\201\110\317\153\265\042\054\010\203\375\323\325\317\073\055\375
+\265\111\220\133\366\255\115\023\312\336\323\246\235\123\121\161
+\143\106\370\112\026\134\230\356\055\155\232\026\241\166\220\342
+\140\103\231\326\211\326\154\056\172\230\262\013\003\054\343\172
+\117\307\335\343\314\343\112\152\215\171\122\372\364\301\257\056
+\217\052\010\313\033\051\202\222\162\103\274\316\210\251\252\247
+\212\121\103\125\205\232\067\003\170\223\310\360\275\264\101\310
+\007\102\232\313\065\227\172\212\201\145\336\035\124\010\001\361
+\144\134\267\027\032\121\274\036\303\131\207\166\030\026\230\356
+\277\366\147\201\213\006\065\305\113\155\131\031\307\322\306\110
+\276\156\024\050\203\112\020\234\033\365\157\274\251\216\365\151
+\376\262\301\125\314\347\024\311\371\133\024\123\121\007\352\316
+\075\344\117\050\037\074\141\011\327\063\322\156\247\156\324\307
+\023\011\157\153\135\024\356\235\211\033\245\152\362\366\370\320
+\162\216\352\162\037\057\064\152\051\012\305\012\354\034\100\205
+\022\367\246\245\323\117\255\300\205\214\114\174\163\040\314\123
+\030\361\262\130\114\001\365\277\352\144\325\134\071\305\316\154
+\314\123\132\126\272\101\017\045\337\153\120\266\307\212\240\275
+\002\302\305\073\125\245\262\144\042\204\121\050\126\256\061\356
+\136\373\013\026\115\106\005\221\200\104\355\254\155\360\127\250
+\372\353\141\110\240\313\033\263\037\216\315\305\041\167\003\204
+\036\374\254\243\103\010\143\214\355\371\047\357\264\260\135\147
+\326\117\355\320\213\076\135\133\311\221\275\226\002\204\075\305
+\115\274\102\077\164\375\074\135\254\134\110\066\136\207\061\057
+\030\154\304\150\356\241\213\311\131\320\030\343\000\200\263\124
+\047\056\231\360\025\123\002\003\001\000\001\243\202\002\017\060
+\202\002\013\060\035\006\003\125\035\016\004\026\004\024\064\161
+\321\070\327\025\066\203\107\153\327\067\144\102\073\216\215\122
+\235\253\060\201\361\006\003\125\035\043\004\201\351\060\201\346
+\200\024\064\161\321\070\327\025\066\203\107\153\327\067\144\102
+\073\216\215\122\235\253\241\201\302\244\201\277\060\201\274\061
+\013\060\011\006\003\125\004\006\023\002\125\123\061\020\060\016
+\006\003\125\004\010\023\007\111\156\144\151\141\156\141\061\025
+\060\023\006\003\125\004\007\023\014\111\156\144\151\141\156\141
+\160\157\154\151\163\061\050\060\046\006\003\125\004\012\023\037
+\123\157\146\164\167\141\162\145\040\151\156\040\164\150\145\040
+\120\165\142\154\151\143\040\111\156\164\145\162\145\163\164\061
+\023\060\021\006\003\125\004\013\023\012\150\157\163\164\155\141
+\163\164\145\162\061\036\060\034\006\003\125\004\003\023\025\103
+\145\162\164\151\146\151\143\141\164\145\040\101\165\164\150\157
+\162\151\164\171\061\045\060\043\006\011\052\206\110\206\367\015
+\001\011\001\026\026\150\157\163\164\155\141\163\164\145\162\100
+\163\160\151\055\151\156\143\056\157\162\147\202\011\000\350\216
+\266\311\370\052\024\050\060\017\006\003\125\035\023\001\001\377
+\004\005\060\003\001\001\377\060\021\006\011\140\206\110\001\206
+\370\102\001\001\004\004\003\002\000\007\060\011\006\003\125\035
+\022\004\002\060\000\060\056\006\011\140\206\110\001\206\370\102
+\001\015\004\041\026\037\123\157\146\164\167\141\162\145\040\151
+\156\040\164\150\145\040\120\165\142\154\151\143\040\111\156\164
+\145\162\145\163\164\060\060\006\011\140\206\110\001\206\370\102
+\001\004\004\043\026\041\150\164\164\160\163\072\057\057\143\141
+\056\163\160\151\055\151\156\143\056\157\162\147\057\143\141\055
+\143\162\154\056\160\145\155\060\062\006\011\140\206\110\001\206
+\370\102\001\003\004\045\026\043\150\164\164\160\163\072\057\057
+\143\141\056\163\160\151\055\151\156\143\056\157\162\147\057\143
+\145\162\164\055\143\162\154\056\160\145\155\060\041\006\003\125
+\035\021\004\032\060\030\201\026\150\157\163\164\155\141\163\164
+\145\162\100\163\160\151\055\151\156\143\056\157\162\147\060\016
+\006\003\125\035\017\001\001\377\004\004\003\002\001\006\060\015
+\006\011\052\206\110\206\367\015\001\001\005\005\000\003\202\002
+\001\000\264\315\275\340\271\352\262\003\053\176\062\351\336\162
+\077\311\113\202\136\235\342\257\125\011\242\014\124\350\317\030
+\074\050\040\035\251\273\003\002\057\122\071\042\371\027\317\255
+\147\220\263\003\177\330\025\343\153\176\273\233\126\001\257\065
+\324\332\271\307\147\027\233\324\325\016\067\263\040\101\056\014
+\001\304\133\371\145\076\302\141\350\322\360\152\225\160\303\306
+\157\325\065\244\254\131\162\341\211\337\241\240\235\044\275\051
+\171\351\141\052\331\323\036\311\106\244\010\170\101\222\162\017
+\253\024\165\355\011\360\242\360\134\357\303\012\142\040\267\302
+\050\146\256\114\057\056\217\105\143\046\226\360\356\061\346\213
+\125\233\252\072\371\202\071\035\210\074\342\007\165\032\341\017
+\261\060\274\161\062\322\072\376\372\241\211\363\103\054\326\162
+\304\171\247\025\110\005\300\330\055\162\002\343\313\075\026\152
+\272\311\270\021\020\342\111\205\314\226\107\140\005\045\056\357
+\165\131\063\365\107\031\026\357\332\154\137\007\310\246\120\266
+\035\313\146\064\045\374\146\203\353\305\266\060\101\370\106\104
+\142\250\301\014\124\346\352\114\132\050\346\256\306\267\376\177
+\073\226\250\056\356\307\150\076\335\000\075\051\257\052\143\253
+\137\356\111\052\055\305\334\373\321\306\323\321\227\126\122\206
+\266\224\353\324\140\121\267\374\036\233\314\002\233\324\037\217
+\371\112\217\266\056\050\073\027\314\305\246\005\343\322\323\265
+\306\003\311\341\110\102\233\313\077\344\027\340\376\015\001\225
+\011\272\270\015\161\344\011\160\167\102\330\115\341\102\251\140
+\203\327\027\211\103\322\324\335\247\030\266\253\324\044\045\207
+\265\324\342\374\056\042\151\275\255\150\054\377\162\265\230\252
+\006\234\347\052\152\270\241\223\166\316\260\363\177\234\341\340
+\117\270\330\206\106\245\063\002\054\045\141\067\052\222\310\254
+\201\164\150\143\207\063\166\275\005\177\136\325\325\002\155\275
+\257\377\052\132\252\111\354\230\171\107\123\221\366\016\064\132
+\311\245\306\353\262\343\305\254\266\240\160\065\273\310\121\151
+\320\362\265\242\062\156\274\077\240\067\071\174\161\066\246\005
+\337\014\022\344\026\247\305\326\313\143\243\225\160\077\346\004
+\243\140
+END
+
+# Trust for Certificate "Software in the Public Interest, Inc. Root CA (2008)"
+CKA_CLASS CK_OBJECT_CLASS CKO_NETSCAPE_TRUST
+CKA_TOKEN CK_BBOOL CK_TRUE
+CKA_PRIVATE CK_BBOOL CK_FALSE
+CKA_MODIFIABLE CK_BBOOL CK_FALSE
+CKA_LABEL UTF8 "Software in the Public Interest, Inc. Root CA (2008)"
+CKA_CERT_SHA1_HASH MULTILINE_OCTAL
+\257\160\210\103\203\202\002\025\315\141\306\274\354\375\067\044
+\251\220\103\034
+END
+CKA_CERT_MD5_HASH MULTILINE_OCTAL
+\052\107\237\140\273\203\164\157\001\003\327\013\015\366\015\170
+END
+CKA_ISSUER MULTILINE_OCTAL
+\060\201\274\061\013\060\011\006\003\125\004\006\023\002\125\123
+\061\020\060\016\006\003\125\004\010\023\007\111\156\144\151\141
+\156\141\061\025\060\023\006\003\125\004\007\023\014\111\156\144
+\151\141\156\141\160\157\154\151\163\061\050\060\046\006\003\125
+\004\012\023\037\123\157\146\164\167\141\162\145\040\151\156\040
+\164\150\145\040\120\165\142\154\151\143\040\111\156\164\145\162
+\145\163\164\061\023\060\021\006\003\125\004\013\023\012\150\157
+\163\164\155\141\163\164\145\162\061\036\060\034\006\003\125\004
+\003\023\025\103\145\162\164\151\146\151\143\141\164\145\040\101
+\165\164\150\157\162\151\164\171\061\045\060\043\006\011\052\206
+\110\206\367\015\001\011\001\026\026\150\157\163\164\155\141\163
+\164\145\162\100\163\160\151\055\151\156\143\056\157\162\147
+END
+CKA_SERIAL_NUMBER MULTILINE_OCTAL
+\002\011\000\350\216\266\311\370\052\024\050
+END
+CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NETSCAPE_TRUSTED_DELEGATOR
+CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NETSCAPE_TRUSTED_DELEGATOR
+CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NETSCAPE_TRUSTED_DELEGATOR
+CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
